docs/src/chapters/dalsze-plany-rozwoju.adoc

```diff
@@ -1 +1,48 @@
+ten proces w ramach procesu CI/CD. Ostatnią brakującą funkcjonalnością jest przesyłanie zakupionych tokenów na adres
+email użytkownika. Już w karcie wymagań projektu zadanie było oznaczone jako opcjonalne, więc odpowiedni wpis nie
+znalazł się w rejestrze. Planowano jego realizację tylko w przypadku ukończenia projektu długo przed terminem, co nie
+miało miejsca. Wszystkie z wymienionych nieukończonych funkcjonalności będą ukończone w najbliższym czasie i zostaną
+uwzględnione w następnym wydaniu.
+
+Aplikacja ma duży potencjał do rozwoju. Nowe funkcjonalności można dodawać praktycznie bez końca. Projekt powstał
+jako połączenie dwóch innych projektów, które miały powstać w ramach projektu _Ingenious_. Był to serwer do przyjmowania
+płatności wraz z generowaniem tokenów oraz portfel multisig z aplikacją mobilną do podpisywania transakcji.
+Naturalną rzeczą jest, że połączenie dwóch projektów skutkowało zmniejszeniem liczby funkcjonalności obu z nich.
+Wiele z nich planowane jest na przyszłe wydania. Moduł płatności powinien być bardziej uniwersalny. Jedną z możliwości
+jest dodanie funkcjonalności pozwalającej na dostosowanie stylu aplikacji przez administratorów. Może to się odbywać
+przykładowo poprzez kaskadowe arkusze stylów CSS. Dobrym pomysłem dyskutowanym na początku było stworzenie serwera z
+wykorzystaniem architektury mikroserwisów. Dzięki temu klienci mogliby uruchomić tylko część serwera odpowiedzialną za
+płatności, wykorzystując stworzone rozwiązanie tylko i wyłącznie jaki bramki płatności. Prawdopodobnie dodanie tej
+funkcjonalności zwiększyłoby szansę na komercjalizację, gdyż pozwoliłoby to na integrację z istniejącymi już sklepami
+internetowymi. Bardzo wiele funkcjonalności można dodać do części odpowiedzialnej za portfel oraz transakcje.
+A aplikacji mobilnej brakuje zarządzania kluczami. W planach jest implementacja odzyskiwania klucza z _mnemonic phase_,
+a także możliwość importu oraz eksportu pliku z kluczem. Można również pomyśleć nad implementacją części funkcjonalności
+aplikacji internetowej w aplikacji mobilnej, takich jak podgląd środków w portfelu czy też przeglądania historii
+płatności. Planowane jest również rozwinięcie zarządzania kanałami. W tym celu planowany jest nowy panel wyświetlający
+szczegóły aktualnie otwartych kanałów. Administrator będzie mógł zamykać tylko wybrane przez siebie kanały, a nie
+wszystkie jednocześnie jak to jest zaimplementowane obecnie. Dodatkowo chciano dodać możliwość skorzystania z funkcji
+_LND_ o nazwie _Autopilot_. Pozwala ona na automatyczne otwieranie kanałów z popularnymi węzłami. Zmniejszyłoby to
+ilość użytkowników, którzy muszą za każdym razem samemu otwierać kanały. Byłoby to szczególnie przydatne na serwerach
+z dużą ilością użytkowników. Gdyż większość użytkowników sieci Lightning miałoby cały czas pośrednio otwarty kanał z
+serwerem. Niestety te funkcjonalności planowane są na daleką przyszłość, o ile kiedykolwiek zostaną zrealizowane.
```

Comment:
trochę się kłóci z 
"Nieważne czy projekt ma szanse na komercjalizację,
czy nie, prawdopodobnie dalej będzie on rozwijany jako projekt poboczny jego autorów."

Reply:
wywaliłem to zdanie